From: "Jinming Wu, Patrick" <me@patrickwu.space>
Date: Wed, 8 Apr 2020 09:57:13 +0800
Subject: [PATCH] Fix empty ~/.config/wslu/baseexec

Previous versions of wslu could create an empty file and this is not
a valid configuration.
---
 src/wslu-header | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/wslu-header b/src/wslu-header
index 8b177da..1eb17d0 100644
--- a/src/wslu-header
+++ b/src/wslu-header
@@ -138,7 +138,7 @@ if [ ! -f ~/.config/wslu/oemcp ]; then
 fi
 
 # generate base exe locaiton
-if [ ! -f ~/.config/wslu/baseexec ]; then
+if [ ! -f ~/.config/wslu/baseexec ] || [ -z "$(cat ~/.config/wslu/baseexec)" ]; then
 	# if it is not generated
 	baseexec_gen
 elif grep -q "/" ~/.config/wslu/baseexec; then
